tools/lkl/lib/hijack/hijack.c

```diff
@@ -283,3 +283,20 @@ int epoll_ctl(int epollfd, int op, int fd, struct epoll_event *event)
 
 	return lkl_call(__lkl__NR_epoll_ctl, 4, epollfd, op, fd, event);
 }
+
+int eventfd(unsigned int count, int flags)
+{
+	return lkl_sys_eventfd2(count, flags);
```

Comment:
it would be good to do like same as other hijacked calls to handle host fds (below is the example of ```setsockopt()```:

```
	CHECK_HOST_CALL(getsockopt);
	if (!is_lklfd(fd))
		return host_setsockopt(fd, level, optname, optval, optlen);
```